headers/libexperimentation_client.h

```diff
@@ -5,22 +5,22 @@
 
 typedef struct Arc_Client Arc_Client;
 
-int last_error_length(void);
+int exp_last_error_length(void);
```

Comment:
@mahatoankitkumar - can we use `expt` as the prefix for experimentation - that seems a little more standard and is also a little less ambiguous.

https://www.merriam-webster.com/dictionary/expt

Reply:
Yeah it's fine, but https://www.merriam-webster.com/dictionary/exp.

Reply:
The ambiguity is explicit in that page 😄